src/bitcoin-cli.cpp

```diff
@@ -437,6 +438,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
         if (conn_type == "block-relay-only") return "block";
         if (conn_type == "manual" || conn_type == "feeler") return conn_type;
         if (conn_type == "addr-fetch") return "addr";
+        if (conn_type == "fullrbf") return "full-rbf";
```

Comment:
This gets kinda ugly: "fullrbf" is also "outbound-full-relay", (well, so is "manual").
Perhaps "outbound-full-relay" should be renamed to "generic-outbound-full-relay" or something, I dunno.